sys/include/net/gnrc/netif/lorawan.h

```diff
@@ -46,6 +46,7 @@ typedef struct {
     uint8_t port;                           /**< LoRaWAN port for the next transmission */
     uint8_t ack_req;                        /**< Request ACK in the next transmission */
     uint8_t otaa;                           /**< whether the next transmission is OTAA or not */
+    bool adr;                               /**< whether ADR is enabled or not */
```

Comment:
As discussed offline, please avoid adding state variables that also need to be kept in `gnrc_lorawan`. Otherwise you will have the same variables twice (one in the interface, one in the node).
Ideally we should directly ask the MAC (e.g mlme_get)

Reply:
ping

Reply:
Done